envd.Dockerfile

```diff
@@ -1,7 +1,12 @@
-FROM rust:bullseye as builder
+FROM quay.io/pypa/manylinux2014_x86_64 as builder
```

Comment:
nit: We'd better add a comments here like "support glibc 2.17".
Otherwise, it's hard to learn from the Dockerfile itself why we're using an outdated base image.